DQM/HLTEvF/plugins/LumiMonitor.cc

```diff
@@ -181,44 +176,44 @@ void LumiMonitor::bookHistograms(DQMStore::IBooker& booker,
                                lumi_binning_.xmin,
                                lumi_binning_.xmax);
   me->setAxisTitle("LS", 1);
-  me->setAxisTitle("scal inst lumi E30 [Hz cm^{-2}]", 2);
+  me->setAxisTitle("online inst lumi E30 [Hz cm^{-2}]", 2);
   histograms.lumiVsLS = me;
 
   me = booker.bookProfile("puVsLS",
-                          "scal PU vs LS",
+                          "online PU vs LS",
                           ls_binning_.nbins,
                           ls_binning_.xmin,
                           ls_binning_.xmax,
                           pu_binning_.xmin,
                           pu_binning_.xmax);
   me->setAxisTitle("LS", 1);
-  me->setAxisTitle("scal PU", 2);
+  me->setAxisTitle("online PU", 2);
   histograms.puVsLS = me;
 }
 
 void LumiMonitor::dqmAnalyze(edm::Event const& event,
                              edm::EventSetup const& setup,
                              Histograms const& histograms) const {
-  int ls = event.id().luminosityBlock();
-
-  float scal_lumi = -1.;
-  float scal_pu = -1.;
-  edm::Handle<LumiScalersCollection> lumiScalers;
-  event.getByToken(lumiScalersToken_, lumiScalers);
-  if (lumiScalers.isValid() and not lumiScalers->empty()) {
-    auto scalit = lumiScalers->begin();
-    scal_lumi = scalit->instantLumi();
-    scal_pu = scalit->pileup();
-  } else {
-    scal_lumi = -1.;
-    scal_pu = -1.;
+  int const ls = event.id().luminosityBlock();
+
+  float online_lumi = -1.f;
+  float online_pu = -1.f;
+  auto const lumiScalersHandle = event.getHandle(lumiScalersToken_);
+  auto const onlineMetaDataDigisHandle = event.getHandle(onlineMetaDataDigisToken_);
+  if (lumiScalersHandle.isValid() and not lumiScalersHandle->empty()) {
+    auto const scalit = lumiScalersHandle->begin();
+    online_lumi = scalit->instantLumi();
+    online_pu = scalit->pileup();
+  } else if (onlineMetaDataDigisHandle.isValid()) {
+    online_lumi = onlineMetaDataDigisHandle->instLumi();
+    online_pu = onlineMetaDataDigisHandle->avgPileUp();
   }
```

Comment:
The logic of this choice was inspired by
https://github.com/cms-sw/cmssw/blob/d03a9e7d44607a324abe7fb94515616fc4efa180/DQM/TrackerCommon/plugins/DetectorStateFilter.cc#L151

and then I noticed it is basically the same as
https://github.com/cms-sw/cmssw/blob/67d902255a04110c7e5b0bc74bd00c9009962d08/CalibTracker/SiStripCommon/plugins/ShallowEventDataProducer.cc#L67